tup-ui/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import './index.css';
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
```

Comment:
_If this code is straight from a starter app, then feel free to ignore this comment. I am new to Typescript._

Non-null operator seems to sweep bug under rug. If we forget to add required #root, shouldn't we let app fail?

Reply:
This is from the boilerplate app.